hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -115,13 +127,13 @@
     // Currently this is a coarse grained aggregation of the bytes/send received.
     // In the future you probably want to split this up in member and client and potentially
     // wan specific.
-    @Probe
+    @Probe(name = NETWORKING_METRIC_NIO_NETWORKING_BYTES_SEND)
```

Comment:
How about setting the `unit = BYTES` attribute for this and the next one?

Reply:
Done